test/integration/Dockerfile

```diff
@@ -1,14 +1,5 @@
 # Dockerfile to test gen_rpc via proper distributed nodes
```

Comment:
Simplified, simply. Works with `make integration` if you're curious.

Reply:
👍